README.md

```diff
@@ -39,7 +39,12 @@ Usage: chart_test.sh <options>
     --verbose         Display verbose output
     --no-lint         Skip chart linting
     --no-install      Skip chart installation
+    --chart-all       Lint/install all charts
```

Comment:
I did add to `chartlib::validate_chart` function `echo "Skipping version increment check..."`.
As printing it before the environment details are displayed won't work, as values wasn't changed for `CHECK_VERSION_INCREMENT=true`, because the `export CHECK_VERSION_INCREMENT=false` was moved to a different place by @unguiculus request.